website_docs/exporting_data.md

```diff
@@ -62,19 +62,55 @@ resources := resource.New(context.Background(),
 )
 ```
 
-## OTLP Exporter
+## OTLP endpoint
 
-OpenTelemetry Protocol (OTLP) export is available in the `go.opentelemetry.io/otel/exporters/otlp/otlptrace` and `go.opentelemetry.io/otel/exporters/otlp/otlpmetric` packages.
+To send trace data to an OTLP endpoint (like the [collector](/docs/collector) or
+Jaeger) you'll want to configure an OTLP exporter that sends to your endpoint.
```

Comment:
Since I think only the most recent versions of jaeger support OTLP, should we mention the minimum required version?

Reply:
Added. Jaeger support OTLP after release [v1.35.0](https://github.com/jaegertracing/jaeger/releases/tag/v1.35.0)
Thanks for reminding